assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -141,6 +148,7 @@ function MediaPane(props) {
   }));
 
   const [isPermissionDialogOpen, setIsPermissionDialogOpen] = useState(false);
+  const [isSearching, setIsSearching] = useState(false);
```

Comment:
Is `useState()` necessary for this?

Can't we just do `const isSearching = Boolean(value.trim());`

Reply:
Got it